frontend/website/src/InvestorsSection.re

```diff
@@ -26,14 +26,24 @@ module Investor = {
               media(twoColumnMedia, [marginLeft(`rem(3.))]),
             ])
           )>
-          <img src=imageSrc className=TeamSection.iconStyle />
-          <h4
-            className=Css.(
-              merge([Style.Body.basic, style([color(Style.Colors.slate)])])
-            )>
-            {ReasonReact.string(name)}
-          </h4>
-        </div>,
+          // Note: change this alt text if we ever hide the investor name
+
+            <img
+              src=imageSrc
+              className=TeamSection.iconStyle
+              alt=""
```

Comment:
maybe say `"Logo image of investor " ++ X`?

Reply:
https://www.w3.org/TR/html51/semantics-embedded-content.html#alt-text example 58

Reply:
spec quoting, I love it